### ISSUE TITLE
Allow multiple pseudo-random signatures

### ISSUE THREAD

**Issue by soytakyon** [SoylentNews/rehash#424]
https://soylentnews.org/~fyngyrz/journal/2940

> I have written many, many sigs; and I've kept them all around in a textfile.
> 
> I would like to use many sigs here. It would be wonderful if the sig part of the account allowed for multiple sigs. It should be trivial to write such a thing — even in Perl, shudder — and it would be a fun enhancement to the site.
> 
> You just need storage of multiple lines of HTML, separated by some obvious token like a linefeed. Actual line breaks in sigs would be <BR> as per usual. Obvious user account options to sequence in order (requires additional storage of one integer), or presented pseudo-randomly. Total length of any one sig could still be limited (although I'd sure like that limit to increase a bit, or at least not count HTML tags against it.)
> 
> Bonus: it'd be way better than That Other Site, and more fun. Those people are so mired in their own egos and incompetence it'd be/take years, or until never, for them to catch up. That goes for all Soylent improvements anyway.
> 
> Pseudocode:
> 
> Split the field by \n
> Chose one of the splits
> Triage the length
> Profit!

**Comment by TheMightyBuzzard** [comment]
Interesting idea. It would take expanding the sig field in the db to a TEXT instead of a VARCHAR but that's not difficult.

**Comment by marty-b** [comment]
Agreed it is an interesting idea.  Beware, though, that invalid HTML constructs are unceremoniously discarded or 'fixed'.

This:
   <marquee>Hee Hee Hee!</marquee>
Becomes:
   Hee Hee Hee!

This:
   <sup>superscripted text
Becomes:
   <sup>superscripted text</sup>

And... these changes are not apparent until after it is saved and one loads the user's preferences again and looks at it, very closely.  It's easy in these (contrived) examples... not so much if you have a large, multi-entry sig.  IOW, may not be very user-friendly at the very time one most needs it to be.

**Comment by fyngyrz** [comment]
So perhaps a little "test sigs" button that opens a tab/page with the sigs, one after another, as they are after being run through the filter?

**Comment by TheMightyBuzzard** [comment]
Meh, be easier to just display all the sigs, one per line, in a div below the box after submission.

**Comment by fyngyrz** [comment]
Any progress on this? You have me all excited about it. 👍

**Comment by TheMightyBuzzard** [comment]
Nah, I'm currently procrastinating about fixing dev which has to be done before martyb can finish testing this release which has to be finished before we start on new stuff.

**Comment by ericgable** [comment]
Hey takyon, what about adding a signature pick list to your **excellent** Tampermonkey script?
